docs/dotnet/dotnet-signalr-metrics.md

```diff
@@ -8,60 +8,109 @@ This document defines semantic conventions for SignalR metrics emitted by .NET c
 
 <!-- toc -->
 
-- [Metric: `signalr_http_transport.connection.duration`](#metric-signalr_http_transportconnectionduration)
-- [Metric: `signalr_http_transport.active_connections`](#metric-signalr_http_transportactive_connections)
+- [Metric: `signalr.http.server.connection.duration`](#metric-signalrhttpserverconnectionduration)
+- [Metric: `signalr.http.server.active_connections`](#metric-signalrhttpserveractive_connections)
 
 <!-- tocstop -->
 
-## Metric: `signalr_http_transport.connection.duration`
+## Metric: `signalr.http.server.connection.duration`
```

Comment:
I think http transport may have had a specific meaning but I need to do more research. Sorry James might have known this stuff offhand but I have to dig through the implementation and conversation history to get the context for all of these which makes each proposed change time consuming. Please still bring it up, thats just why I don't have quick answers for everything.

Reply:
I did do more research here and I think this is the context:

- SignalR is defined as a protocol that can operate over any reliable, in-order, duplex, message based transport
- Right now the only implementations of transports all use http, but in three different variants (long polling, server sent events, and web sockets)
- The Micrsoft.AspNetCore.Http.Connections assembly these metrics are implemented inside of has the logic to create those various http transport flavors.
- It feels reasonable to omit the word transport and let http imply 'http-based transport'. I don't see any other likely interpretation that would create ambiguity.